docs/user/mobile-applications.rst

```diff
@@ -50,18 +50,22 @@ The Android application supports the following features:
 Installation
 ^^^^^^^^^^^^
 
-The Android application is currently only available in beta state and can be downloaded over `Firebase App Distribution <https://appdistribution.firebase.dev/i/f5dedbb0fc6dc0da>`_. Make sure to follow the steps described by Firebase App Distribution to install the application.
+The Android application is currently only available in beta state and can be downloaded over `Firebase App Distribution <https://appdistribution.firebase.dev/i/f5dedbb0fc6dc0da>`_.
```

Comment:
Shouldn't that be iOS?

Reply:
Formatting the document got in the way :)